@here/harp-mapview/lib/Utils.ts

```diff
@@ -685,6 +663,7 @@ export namespace MapViewUtils {
                   TERRAIN_ZOOM_LEVEL
               )
             : undefined;
+        const final = !elevationProvider || elevation !== undefined;
```

Comment:
Minor, elevation is undefined if !evelationProvider is undefined.

Reply:
If elevationProvider is undefined, then !elevationProvider is true and the expression is shortcircuited, resulting in final === true.
I don't see the problem.

Reply:
Sorry, I wasn't clear, line 660 already does the check for `elevationProvider`, and if it is undefined, then `elevation` is undefined, hence you can use just `const final = elevation !== undefined;`

Reply:
@atomicsulfate , will you change this or keep it? It is a minor, if you prefer not to change, then I will just approve.

Reply:
> Sorry, I wasn't clear, line 660 already does the check for `elevationProvider`, and if it is undefined, then `elevation` is undefined, hence you can use just `const final = elevation !== undefined;`

If elevationProvider is undefined, final should be true, cause that means we are not using elevation and therefore the computed target is final.
final = elevation !== undefined  <- That's only applicable if there's an elevation provider.

Reply:
Ahh, I missed that it should be true if it is undefined, because no better result will come, ok will approve.